docs/demo/demo.js

```diff
@@ -7,30 +7,44 @@ if (!window.fetch) {
   window.fetch = unfetch;
 }
 
+onunhandledrejection = function (e) {
+  throw e.reason;
+};
+
 var $markdownElem = document.querySelector('#markdown');
+var $markedVerElem = document.querySelector('#markedVersion');
+var $commitVerElem = document.querySelector('#commitVersion');
+$commitVerElem.style.display = 'none';
+var $markedVer = document.querySelector('#markedCdn');
 var $optionsElem = document.querySelector('#options');
 var $outputTypeElem = document.querySelector('#outputType');
 var $inputTypeElem = document.querySelector('#inputType');
+var $responseTimeElem = document.querySelector('#responseTime');
+var $previewElem = document.querySelector('#preview');
 var $previewIframe = document.querySelector('#preview iframe');
 var $permalinkElem = document.querySelector('#permalink');
 var $clearElem = document.querySelector('#clear');
 var $htmlElem = document.querySelector('#html');
 var $lexerElem = document.querySelector('#lexer');
 var $panes = document.querySelectorAll('.pane');
 var $inputPanes = document.querySelectorAll('.inputPane');
+var lastInput = '';
 var inputDirty = true;
 var $activeOutputElem = null;
-var changeTimeout = null;
 var search = searchToObject();
 
-var iframeLoaded = false;
+var markedVersions = {
+  commit: 'commit',
+  master: 'https://cdn.jsdelivr.net/gh/markedjs/marked/lib/marked.js'
+};
+var markedVersionCache = {};
+
 $previewIframe.addEventListener('load', function () {
-  iframeLoaded = true;
+  lastInput = '';
   inputDirty = true;
-  checkForChanges();
 });
 
-if ('text' in search) {
+if ('text' in search && search.text) {
```

Comment:
Is it necessary to do both checks here?
I would think `if (search.text)` would be sufficient.

Reply:
i changed it